### ISSUE TITLE
editing device group

### ISSUE THREAD

**Issue by MatthiasGreshake** [OsnaCS/drk-funk#231]
editing device group is not possible (should it be possible?)

**Comment by elanfer** [comment]
yep should be possible, just if you got a typo failure you have to edit this

**Comment by elanfer** [comment]
i'm on with this

**Comment by elanfer** [comment]
scheduled for v0.3 removing the button at first for presentation
